NEWS.md

```diff
@@ -68,5 +156,5 @@ of AxoSyslog, contribute.
 
 We would like to thank the following people for their contribution:
 
-Andras Mitzki, Attila Szakacs, Balazs Scheidler, Hofi, László Várady,
-Szilard Parrag, Tamás Kosztyu, shifter, Shiraz McClennon
+Andras Mitzki, Attila Szakacs, László Várady, Ross Williams,
```

Comment:
Let's add Szilárd and Bazsi manually, they review and help a lot.

Reply:
Added